validator/tests/unit3/test_context_manager/tests.py

```diff
@@ -206,3 +290,531 @@ def test_squash_no_updates(self):
         self.assertEqual(
             [StateChange(address='aaa', value=b'xyz', type=StateChange.SET)],
             [c for c in changes])
+
+    @unittest.skip("Support for context manager parallelization")
```

Comment:
Not in this sprint, I don't think, but I made a jira story for making them pass.